packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -5,6 +5,7 @@ import { Price } from '@magento/peregrine';
 import classify from 'src/classify';
 import Button from 'src/components/Button';
 import Carousel from 'src/components/ProductImageCarousel';
+import Options from 'src/components/ProductOptions';
```

Comment:
Can we dynamically import this instead, on demand in the `productOptions` getter?